packages/codemods/src/transforms/v2/web-react/button-text.ts

```diff
@@ -0,0 +1,51 @@
+import { API, FileInfo } from 'jscodeshift';
+
+const transform = (fileInfo: FileInfo, api: API) => {
+  const j = api.jscodeshift;
+  const root = j(fileInfo.source);
+
+  // Find import statements for the specific module and Button specifier
+  const importStatements = root.find(j.ImportDeclaration, {
+    source: {
+      value: (value: string) => /^@lmc-eu\/spirit-web-react(\/.*)?$/.test(value),
```

Comment:
I suppose this expression is yet to be abstracted with future codemods?

Reply:
It is checking the import of the component, and it could be done in multiple ways, so we need to check all of them.